modules/ROOT/pages/redelivery-policy.adoc

```diff
@@ -0,0 +1,63 @@
+= Redelivery Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, flow, redelivery, policy
+
+A redelivery policy is a filter added to Mule sources, such as modules and connectors, to prevent the dispatch of messages known to cause errors.
+
+When a redelivery policy is added to a flow's source, Mule evaluates the received data before it executes the flow's components. If the data caused failures a specified number of times in the past, the redelivery policy prevents the flow from processing the received data and raises a `REDELIVERY_EXHAUSTED` error. This behavior prevents the runtime from wasting resources by limiting the number of times Mule will execute flow messages that cause errors.
```

Comment:
How is the policy added; who does it? Convert to active voice.

Not sure we really need the final sentence. Actually, it is the core of the topic; perhaps it should be incorporated into the initial paragraph: "A redelivery policy helps you conserve valuable resources by limiting the number of times Mule runtime engine executes flow messages that cause errors. The policy acts as a filter...."